forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -226,6 +226,9 @@ pub struct Command {
 }
 
 fn parse_abi_path(s: &str) -> Result<Either<PathBuf, Url>, String> {
```

Comment:
This function may need a re-design; i.e. we most likely will need to read the `contract_id` val